ecosystem/sep-0012.md

```diff
@@ -315,3 +316,35 @@ Situation | Response
 Success | `200 OK`
 Client not authenticated properly | `401 Unauthorized`
 Anchor has no information on the customer | `404 Not Found`
+
+## Customer callback PUT
+
+Allow the wallet to provide a callback URL to the anchor. The provided callback URL will replace (and supercede) any previously-set callback URL for this account.
+
+Whenever the user's `status` field changes, the anchor will issue a POST request to the callback URL. The payload of the POST request will be the same as [`GET /customer`](#customer-get).
+
+It's the wallet's responsibility to send a working callback URL to the anchor. If callback POST fails due to a problem on the callback URL's side, the anchor should not retry it.
```

Comment:
>If callback POST fails due to a problem on the callback URL's side, the anchor should not retry it.

❗I think we should allow retrying, but leave the specifics undefined. Retrying callbacks is a common practice, but we can't be too prescriptive without burdening services that don't have the capabilities to support robust callback retry systems.

By disallowing retries here we encourage clients to build systems that can't handle duplicate requests, which could be dangerous. A callback URL should always handle cases like being called more than once, or other edge cases, and there's little reason to disallow retries when a callback endpoint needs to handle those other cases.

```suggestion
It's the wallet's responsibility to send a working callback URL to the anchor. If callback POST fails due to a problem on the callback URL's side, the anchor may retry it, although is not required to.
```

Alternatively, if we don't feel like we want to make a decision on this just yet, delete this statement and leave retries undefined until such time as they become defined.

```suggestion
It's the wallet's responsibility to send a working callback URL to the anchor.
```